EnglishFilter/sections/specific.txt

```diff
@@ -143,6 +143,7 @@ pornwhite.com,aplayer2.me,viralizeit.net,sex3.com,softonic.com,3pornstarmovies.c
 !
 mangazuki.online##.body-top-ads
 softfamous.com##.container_ads
+gottanut.com##div[class^="reclam-"]
```

Comment:
Change to (search for reclam-text-mobile to see other elements):
[class^="reclam-"]